### ISSUE TITLE
Rename for old germanic languages?

### ISSUE THREAD

**Issue by kylepjohnson** [cltk/probabilistic_models#1]
@Sedictious Will the purpose of these libraries be like those of:

* https://github.com/cltk/greek_models_cltk
* https://github.com/cltk/latin_models_cltk

If so, you can call it `germanic_models_cltk`.

**Comment by Sedictious** [comment]
I made my first commit so you can get an idea on the specific topics addressed. Essentially, the main idea would be to provide easily modifiable source code for models that would help the development of the rune analyzer and maybe re-purposed for germanic-focused tasks

**Comment by kylepjohnson** [comment]
> easily modifiable source code for models

Sure, this is a good idea. I just renamed it `germanic_models_cltk`, however you can change it, if this is currently only for experimentation. Just call it something specific, perhaps `germanic_model_development`.

**Comment by kylepjohnson** [comment]
FYI @Sedictious and @clemsciences I plan on deleting this repo, in preparation of our upcoming api changes. There is no data in here, just a couple simple ML files.

**Comment by Sedictious** [comment]
I agree, as of now it's just noise. We can re-introduce it later if we have any meaningful work to share

**Comment by clemsciences** [comment]
Maybe @Sedictious can become the owner of this repository.

**Comment by kylepjohnson** [comment]
Closing in light of v2 cltk release.
